### PR TITLE
at_onboarding_flutter documentation added and example app updated with at_app template

### PR DESCRIPTION
**Pull request by Intiserahmed**
**- What I did**

**- How I did it**

**- How to verify it**

**- Description for the changelog**

### CODE REVIEW

File: at_onboarding_flutter/example/ios/Runner/Info.plist [hunk 1]
```diff
@@ -4,14 +4,16 @@
 <dict>
 	<key>CFBundleDevelopmentRegion</key>
 	<string>$(DEVELOPMENT_LANGUAGE)</string>
```
Comment (cpswan) [inline at_onboarding_flutter/example/ios/Runner/Info.plist:6]: In at_onboarding_flutter/example/ios/Flutter/AppFrameworkInfo.plist this has changed from `$(DEVELOPMENT_LANGUAGE)` to `en`. Should that be consistent here?

Comment (Intiserahmed) [reply at_onboarding_flutter/example/ios/Runner/Info.plist:6]: It shows the default language english in IOS emulator when i tested. I guess this the official flutter project creation code that gives the flexibility to choose any default language for ios side.

Comment (cpswan) [reply at_onboarding_flutter/example/ios/Runner/Info.plist:6]: My question was less about what it does, and more about whether we should consistently use `$(DEVELOPMENT_LANGUAGE)` or `en` as right now some files have one and some the other?

Comment (Intiserahmed) [reply at_onboarding_flutter/example/ios/Runner/Info.plist:6]: All the flutter project creation has similar inconsistency (In AppFrameworkInfo.plist & this file) but we can have a choice to make it consistent.

Comment (cpswan) [reply at_onboarding_flutter/example/ios/Runner/Info.plist:6]: Sounds like a topic for the eng stand (and maybe the architecture call that follows)

**Review by CynthiaPeter — COMMENTED** [review]
For the features,(Line 18-22) we may need to work on those. 

For example, line  18 and 19 could be: 
- The at_onbaording package takes away the difficulty in implementing atsign authentication.
- Generate and Supports free atsigns.

I am not sure if this best translates to what should be, but this is simpler to understand, I hope. What do you think @gkc @cpswan

**Pull request closed without merging**